### PR TITLE
docs: DOC-1517 updated oop diagram

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR updates the airgap vSphere install order. Feedback from the field points to confusion about what OVA to start with. We can improve and prevent this by changing the order of operations always start with the airgap support OVA. 

Other changes include updating the change in airgap binary containing packs and images. It needs to be downloaded now into the airgap support VM. 

The last change is to update the Palette CLI install guides and add the new prompt for template folder. 

## Changed Pages

💻 [VerteX vSphere Airgap Install](https://deploy-preview-5033--docs-spectrocloud.netlify.app/enterprise-version/install-palette/install-on-vmware/airgap-install/environment-setup/vmware-vsphere-airgap-instructions/)
💻 [Palette vSphere Airgap Install](https://deploy-preview-5033--docs-spectrocloud.netlify.app/vertex/install-palette-vertex/install-on-vmware/airgap-install/environment-setup/vmware-vsphere-airgap-instructions/)

## Jira Tickets

🎫 [DOC-1517](https://spectrocloud.atlassian.net/browse/DOC-1517)

## Backports

Can this PR be backported?

- [x] Yes. _Remember to add the relevant backport labels to your PR._
- [ ] No. _Please leave a short comment below about why this PR cannot be backported._

[DOC-1517]: https://spectrocloud.atlassian.net/browse/DOC-1517?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/enterprise-version/install-palette/install-on-vmware/airgap-install/environment-setup/vmware-vsphere-airgap-instructions.md [hunk 1]
```diff
@@ -152,22 +107,22 @@ The default container runtime for OVAs is [Podman](https://podman.io/), not Dock
...
     dependencies required to deploy a Palette. Once the deployment is complete, the airgap support VM is displayed in
     the vSphere inventory. The VM will be powered off. Power on the VM to continue.
 
-14. SSH into the airgap support VM. Use the following command to SSH into the VM. Replace the IP address below with the
+11. SSH into the airgap support VM. Use the following command to SSH into the VM. Replace the IP address below with the
     IP address or hostname of the airgap support VM. The default user account is `ubuntu`. Replace the path to the
     private SSH key and the IP address with the IP address or domain name of the airgap support VM.
 
     ```shell
     ssh -identity_file /path/to/private/key ubuntu@palette.example.com
```
Comment (benradstone) [inline docs/docs-content/enterprise-version/install-palette/install-on-vmware/airgap-install/environment-setup/vmware-vsphere-airgap-instructions.md:122]:
```suggestion
    ssh --identity_file /path/to/private/key ubuntu@palette.example.com
```

File: docs/docs-content/vertex/install-palette-vertex/install-on-vmware/airgap-install/environment-setup/vmware-vsphere-airgap-instructions.md [hunk 2]
```diff
@@ -157,22 +111,22 @@ The default container runtime for OVAs is [Podman](https://podman.io/), not Dock
...
     dependencies required to deploy a VerteX. Once the deployment is complete, the airgap support VM is displayed in the
     vSphere inventory. The VM will be powered off. Power on the VM to continue.
 
-14. SSH into the airgap support VM. Use the following command to SSH into the VM. Replace the IP address below with the
+11. SSH into the airgap support VM. Use the following command to SSH into the VM. Replace the IP address below with the
     IP address or hostname of the airgap support VM. The default user account is `ubuntu`. Replace the path to the
     private SSH key and the IP address with the IP address or domain name of the airgap support VM.
 
     ```shell
     ssh -identity_file /path/to/private/key ubuntu@vertex.example.com
```
Comment (benradstone) [inline docs/docs-content/vertex/install-palette-vertex/install-on-vmware/airgap-install/environment-setup/vmware-vsphere-airgap-instructions.md:126]:
```suggestion
    ssh --identity_file /path/to/private/key ubuntu@vertex.example.com
```

File: docs/docs-content/vertex/install-palette-vertex/install-on-vmware/airgap-install/environment-setup/vmware-vsphere-airgap-instructions.md [hunk 3]
```diff
@@ -371,26 +325,26 @@ The default container runtime for OVAs is [Podman](https://podman.io/), not Dock
...
 
     ```shell
-    chmod +x airgap-vertex-<version>.bin && ./airgap-vertex-<version>.bin
+    curl --user XXXXX:YYYYY https://software-private.spectrocloud.com/airgap/thirdparty/airgap-thirdparty-X.X.X.bin \
+    --output airgap-upload.bin
     ```
 
-    Consider the following example for reference.
+20. Use the following command to start the third-party binary. The third-party binary uploads the release-specific packs
+    and images to the registry configured in step **17** of this guide. This process may take some time to complete.
```
Comment (benradstone) [inline docs/docs-content/vertex/install-palette-vertex/install-on-vmware/airgap-install/environment-setup/vmware-vsphere-airgap-instructions.md:342]: I may be misinterpreting this, but does the user configure the registry in step 17?

I do see the registries displayed in the example output, but I can't seem to find the step when the registry is configured.

File: docs/docs-content/enterprise-version/install-palette/install-on-vmware/airgap-install/environment-setup/vmware-vsphere-airgap-instructions.md [hunk 4]
```diff
@@ -366,25 +321,26 @@ The default container runtime for OVAs is [Podman](https://podman.io/), not Dock
...
 
     ```shell
-    chmod +x airgap-<version>.bin && ./airgap-<version>.bin
+    curl --user XXXXX:YYYYY https://software-private.spectrocloud.com/airgap/thirdparty/airgap-thirdparty-X.X.X.bin \
+    --output airgap-upload.bin
     ```
 
-    Consider the following example for reference.
+20. Use the following command to start the third-party binary. The third-party binary uploads the release-specific packs
+    and images to the registry configured in step **17** of this guide. This process may take some time to complete.
```
Comment (benradstone) [inline docs/docs-content/enterprise-version/install-palette/install-on-vmware/airgap-install/environment-setup/vmware-vsphere-airgap-instructions.md:338]: As mentioned in my comment for the vertex guide:

> I may be misinterpreting this, but does the user configure the registry in step 17?
> 
> I see the registries displayed in the example output, but I can't find the step when the registry is configured.

**Review by benradstone — APPROVED** [review]
Approved with some comments and two suggestions.

File: docs/docs-content/vertex/install-palette-vertex/install-on-vmware/airgap-install/environment-setup/vmware-vsphere-airgap-instructions.md (hunk 3 above)
Comment (karl-cardenas-coding) [reply docs/docs-content/vertex/install-palette-vertex/install-on-vmware/airgap-install/environment-setup/vmware-vsphere-airgap-instructions.md:342]: @benradstone the script is what initializes and configured the Harbor registry.

**Pull request merged**